internal/servers/worker_auth.go

```diff
@@ -2,12 +2,19 @@ package servers
 
 import (
 	"context"
+	"time"
 
 	"github.com/hashicorp/boundary/internal/errors"
 	"github.com/hashicorp/boundary/internal/servers/store"
 	"google.golang.org/protobuf/proto"
 )
 
+const (
+	// DefaultCertificateLifetime represents how long root certificates are valid for.
+	// The default is set to a year
+	DefaultCertificateLifetime = time.Hour * 24 * 365 * 24
```

Comment:
```suggestion
	DefaultCertificateLifetime = time.Hour * 24 * 365
```

Reply:
Don't define this. The root rotation function in the library is specifically designed to not do anything if it's not time to rotate. Have it simply run once an hour or so. If in the future we allow people to modify the default certificate lifetime (which, if we needed to reference it, should really be `nodeenrollment.DefaultCertificateLifetime` anyways) we can simply use fractions of that defined value.

Reply:
I've updated the job to run once an hour, and I've added a FIXME indicating the year cert lifetime is temporary